### PR TITLE
[dy2static] Add log and check predicted scores in test_transformer

### PR DESCRIPTION
**Pull request by liym27**
As the title.

### CODE REVIEW

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py [hunk 1]
```diff
@@ -29,6 +29,7 @@
 place = fluid.CUDAPlace(0) if fluid.is_compiled_with_cuda() else fluid.CPUPlace(
 )
 SEED = 10
+step_num = 10
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py:32]: Usually this kind of constant which won't be changed will be write as `STEP_NUM`

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py [hunk 2]
```diff
@@ -263,12 +265,28 @@ def predict_dygraph(args, batch_generator):
...
+                if step_idx == 0:
+                    logging.info(
+                        "Dygraph Predict: step_idx: %d, 1st seq_id: %d, 1st seq_score: %.2f"
+                        % (step_idx, seq_ids[0][0][0], seq_scores[0][0]))
+                    avg_batch_time = time.time()
+                else:
+                    speed = args.print_step / (time.time() - avg_batch_time)
+                    speed_list.append(speed)
+                    logging.info(
+                        "Dygraph Predict: step_idx: %d, 1st seq_id: %d, 1st seq_score: %.2f, speed: %.3f step/s"
```
Comment (zhhsplendid) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py:280]: Not import English knowledge: "step/s" -> "steps/s"

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py (hunk 1 above)
Comment (liym27) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py:32]: Thanks. I will modify it next PR.

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py (hunk 2 above)
Comment (liym27) [reply python/paddle/fluid/tests/unittests/dygraph_to_static/test_transformer.py:280]: Thanks. I will modify it next PR.

**Pull request merged**